Makefile

```diff
@@ -1,24 +1,41 @@
 .PHONY: build-prod
 build-prod: generate-docs ## Build the container image but with the upstream tag
-	podman build --build-arg INSIGHTS_MCP_VERSION=$(VERSION) --tag ghcr.io/redhatinsights/insights-mcp:latest .
+	podman build \
+	  --build-arg INSIGHTS_MCP_VERSION=$(VERSION) \
+	  --build-arg CONTAINER_BRAND=$(CONTAINER_BRAND) \
+	  --tag ghcr.io/redhatinsights/$(CONTAINER_BRAND)-mcp:latest
```

Comment:
@ochosi oh - you are right - here it's wrong…
I'll fix that

Reply:
Thanks, yes, that's what I meant.